audits/manifest/exists.js

```diff
@@ -47,7 +47,9 @@ class ManifestExists extends Audit {
    */
   static audit(artifacts) {
     return ManifestExists.generateAuditResult(
-      typeof artifacts.manifest !== 'undefined'
+      typeof artifacts.manifest.value !== 'undefined',
+      undefined,
```

Comment:
it's the audit's `rawValue`, whatever that is. It could still be a boolean in this case, but then it'll be displayed twice with the current cli pretty printing.

The only place this is used currently is in the FMP output, which puts the actual time in parentheses. There's probably a better way of piping that through